packages/sdk/src/build/build-authorizations.js

```diff
@@ -1,15 +1,15 @@
-import {pipe, makeAuthorizer} from "../interaction/interaction.js"
+import {pipe, makeAccountRole} from "../interaction/interaction.js"
 
 const roles = {
   authorizer: true,
 }
```

Comment:
Dislike how this solution requires this object to need to have a singular property, the code doesn't really make this very obvious and will probably cause some subtle and confusing issues in the future.

Reply:
Removed and add to opts: { }